src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -74,6 +74,21 @@ public static void Run(ConfigGetArgs args, Func<ILogger> getLogger)
         }
     }
 
+    internal static class ConfigSetRunner
+    {
+        public static void Run(ConfigSetArgs args, Func<ILogger> getLogger)
+        {
+            RunnerHelper.EnsureArgumentsNotNull(args, getLogger);
+            RunnerHelper.ValidateConfigKey(args.ConfigKey);
+            ISettings settings = string.IsNullOrEmpty(args.ConfigFile)
+                ? RunnerHelper.GetSettingsFromDirectory(null)
+                : Settings.LoadSpecificSettings(Path.GetDirectoryName(args.ConfigFile), args.ConfigFile);
```

Comment:
How about using existing utility method `XPlatUtility.ProcessConfigFikle()` to fetch the settings.
https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/XPlatUtility.cs#L72

For example, dotnet nuget verify command uses this utility method https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#LL65C50-L65C67